types/stripe/index.d.ts

```diff
@@ -5,11 +5,12 @@
 //                 Amrit Kahlon <https://github.com/amritk/>
 //                 Adam Cmiel <https://github.com/adamcmiel>
 //                 Justin Leider <https://github.com/jleider>
+//                 Kamil Gałuszka <https://github.com/galuszkak>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
```

Comment:
@RyanCavanaugh but that doesn't resolve problem that we have. 
Stripe encourage to have 2 version of Stripe (v2 and v3), but You can't have 2 version of @types/stripe because package.json allows You have only 1 version of package. 

So basically we can't have 2 version of @types therefore we can't use that for declaration merging and benefit of having both in our IDE.